tests/python_client/testcases/test_search.py

```diff
@@ -3618,20 +3619,19 @@ def test_search_string_field_not_primary_is_empty(self, _async):
 
 
         search_string_exp = "varchar >= \"\""
-        
+
         # 3. search
         log.info("test_search_string_field_not_primary: searching collection %s" % collection_w.name)
         vectors = [[random.random() for _ in range(default_dim)] for _ in range(default_nq)]
         output_fields = [default_string_field_name, default_float_field_name]
         collection_w.search(vectors[:default_nq], default_search_field,
-                            default_search_params, default_limit, 
+                            default_search_params, default_limit,
                             search_string_exp,
                             output_fields=output_fields,
                             _async=_async,
                             travel_timestamp=0,
                             check_task=CheckTasks.check_search_results,
                             check_items={"nq": default_nq,
-                                         "ids": insert_ids,
+                                         "id": insert_ids,
```

Comment:
Here "ids" is our check items in check function, should not be modified as "id".